tox.ini

```diff
@@ -37,11 +54,9 @@ setenv =
     PYTHONPATH = {toxinidir}
 deps =
     -r{toxinidir}/requirements_dev.txt
-    windows: pipwin
     lxml: lxml
 commands =
-    others: pip install -r {toxinidir}/requirements_win.txt
-    windows: pipwin install -r {toxinidir}/requirements_win.txt
+    pip install -r {toxinidir}/requirements_others.txt
```

Comment:
Should this not go to the `deps` section as `-r {toxinidir}/requirements_others.txt`, similar to `-r{toxinidir}/requirements_dev.txt`?

Reply:
Yes, i moved it :)